source/playbooks/AFSBP/ssmdocs/AFSBP_RDS.13.yaml

```diff
@@ -76,7 +76,7 @@ mainSteps:
           Regions: [ '{{ParseInput.RemediationRegion}}' ]
           ExecutionRoleName: '{{RemediationRoleName}}'
       RuntimeParameters:
-        DbiResourceId: '{{ ParseInput.DbiResourceId }}'
+        DBInstanceIdentifier: '{{ ParseInput.DBInstanceIdentifier }}'
```

Comment:
Please mirror any updates to this file in the SC implementation. It also has the control id RDS.13.

It's possible to make the SC implementation the source of truth across different standards, see CIS 1.4 for an example. But I didn't do that for the existing standards yet. I can't think of a reason not to...